scout/constants/acmg.py

```diff
@@ -126,15 +129,16 @@
                     (
                         "PM5",
                         {
-                            "short": "Similar to known pathogenic aa (HQ)",
-                            "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before,  the amino acids have similar properties and the evidence is strong",
+                            "short": "Similar to known pathogenic aa",
+                            "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before, the amino acids have similar properties and the evidence is strong",
```

Comment:
Again, likely "HQ" to mean "evidence is strong". This is somewhat evident from Richards 2015. Let's skip it.